system/translations.rst

```diff
@@ -1,18 +1,91 @@
 Translations
 ************
 
-In Zammad (as admin) you can translate strings having typos or even not being translated.
-You can always "Get last translations" to ensure that you're up to date before doing that.
+Translations of Zammad are processed centrally on our `Weblate instance`_.
+This allows non development people to review and update translation strings of
+the language they actually speak.
 
-**Note:** The language for translation is always the language you chose within your profile.
-As Zammad's main language is English, you can't translate into English.
+Your language is in an incomplete translation state or has typos?
+Please consider helping us sorting this out! All you need is either a GitHub
+account or register directly on our instance.
 
-.. image:: /images/system/translations/1.jpg
+.. _Weblate instance:
+   https://translations.zammad.org/
 
-Translated strings are highlighted within Zammad.
-If you've translated strings, you have two options:
-You can send your changes to our central system, this will share your translation with other users. This is a great contribution.
+   .. hint::
 
-If you don't like your changes or want to start over, you can discard your changes at any time.
+      | Did you know?
+      | This is also where documentation translations are handled. 🤓
 
-.. image:: /images/system/translations/2.jpg
+--------------------------------------------------------------------------------
+
+Apart from these centralized translations, there's still local translations
+you can manage. This is relevant if you added custom objects that require
+translations or even states.
+
+   .. warning::
+
+      You can no longer synchronize these translations upstream like prior
+      Zammad 5.1.
+
+   .. figure:: /images/system/translations/translation-management.png
+      :alt: Translation management page within the admin menu
+
+How do I get the most recent translations?
+------------------------------------------
+
+Because of how Zammad ships translations, the only way to update the
+central translations is to update your Zammad installation. The benefit of
+this is that you no longer need internet access during a Zammad upgrade as
+the package has everything it requires already.
+
+But... I have custom objects?!
+------------------------------
+
+No problem at all! If you can't find the strings of your objects already within
+the translation list, please ensure to `add the translation strings`_ to your
+installation.
+
+Now you can translate your objects as needed.
+
+.. _add the translation strings:
+   https://docs.zammad.org/en/latest/admin/console/other-usefull-commands.html#add-translation
```

Comment:
@MrGeneration btw, it should be `useful` with just one `l`. Maybe you can fix that someday.